pkg/handlers/v1/elb.go

```diff
@@ -28,68 +35,97 @@ func extractELBNetworkInfo(config *elbConfiguration) Change {
 
 type elbTransformer struct{}
 
-func (t elbTransformer) Create(event awsConfigEvent) (Output, error) {
+// Some ELBs have createdTime in the form of milliseconds, which is int64 in Go,
```

Comment:
Nit: It looks like this is a difference in load balancers, so classic load balancers seem to have the millisecond timestamp and the rest have a string. I think I would prefer to handle the timestamp based on that. I'm not super into this raw message stuff, so I'd prefer to avoid it.

Reply:
- [ ] Unmarshal events by load balancer type, instead of raw message parsing